### ISSUE TITLE
If tracing_subscriber is enabled, rocket::debug! messages do not appear in logs, though other log macros work.

### ISSUE THREAD

**Issue by DanielJoyce** [SergioBenitez/Rocket#1544]
Version: 0.5.0-dev on linux

If I set my logging up using the tracing logger:

```
#[rocket::main]
pub async fn main() {
    tracing_subscriber::fmt().try_init().unwrap_or(());
    let rocket = app::assemble_rocket();
    rocket.launch().await.expect("Launch failed!")
}
```

Then I see the tracing style log format, but messages logged with `rocket::debug!` do not appear in the output

If I comment out the tracing_subscriber line, then I do see debug level messages in the normal rocket logger output.

I've tried setting RUST_LOG, ROCK_LOG and ROCKET_LOG_LEVEL env vars to debug, and debug level messages will not show up when tracing_subscriber is uncommeneted.

**Comment by DanielJoyce** [comment]
Error appears to be here:

https://github.com/SergioBenitez/Rocket/blob/8ee581a6978058b0e3228dc4f422126a1d2e80fa/core/lib/src/logger.rs#L41

**Comment by DanielJoyce** [comment]
the log package supports a 'debug' level, so maybe use this instead of 'trace' to avoid confusion

**Comment by SergioBenitez** [comment]
See #21.

**Comment by DanielJoyce** [comment]
Is this confirmed fix? I thought this might be in the 0.5.0-dev branch already but doesn't appear to be, so I filed this in the possibility #21 missed a case.
